### DISCUSSION TITLE
V7 where did the errors field go from the useForm?

### DISCUSSION THREAD

**Discussion by Happy-Child** [react-hook-form/react-hook-form#4594] [Q&A] [answered]
in the 6th version, the `errors` field was obtained from `useForm`

in the 7th he is not there

has it moved to a different location?

thanks

**Comment by jorisre** [comment, marked as answer]
Hi @Happy-Child 👋🏻 

It moved inside `formState` ⬇️ 

[Migration guide](https://react-hook-form.com/migrate-v6-to-v7/)

![Screenshot 2021-04-02 at 21 30 48](https://user-images.githubusercontent.com/7545547/113447791-dd5e8b80-93fa-11eb-8161-71f80425b494.png)
